Octokit.Tests/Clients/ReleasesClientTests.cs

```diff
@@ -51,7 +51,9 @@ public void RequestsCorrectUrlWithApiOptions()
                 var releasesClient = new ReleasesClient(Substitute.For<IApiConnection>());
 
                 await Assert.ThrowsAsync<ArgumentNullException>(() => releasesClient.GetAll(null, "name"));
+                await Assert.ThrowsAsync<ArgumentException>(() => releasesClient.GetAll("", "name"));
                 await Assert.ThrowsAsync<ArgumentNullException>(() => releasesClient.GetAll("owner", null));
+                await Assert.ThrowsAsync<ArgumentException>(() => releasesClient.GetAll("owner", ""));
                 await Assert.ThrowsAsync<ArgumentNullException>(() => releasesClient.GetAll("owner", "name", null));
```

Comment:
We should probably check `("", "name", ApiOptions.None)` and `("owner", "", ApiOptions.None)` as well